oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -28,6 +28,10 @@ def get_port():
 
 
 if __name__ == '__main__':
+    # To avoid systemd in container environment
+    use_systemd = True
+    if len(sys.argv) > 1:
+        use_systemd = not sys.argv[1] == 'no'
```

Comment:
Should the argument being passed in be more descriptive?
It is not clear what ` mcs-loadbrm.py no` would do w/o digging into mcs-loadbrm.py.

Reply:
There should be getopt org ArgParse used here but I don't want to bother with them having a single argument possible. If we have more then we'll need to use one of those modules.

Reply:
There is a comment here already and another one must go into the corresponding CMAPI PR.

Reply:
Okay, if there is one in the CMAPI PR then that's fine.